docs/pt/docs/tutorial/request-forms.md

```diff
@@ -0,0 +1,58 @@
+# Dados do formulário
+
+Quando você precisa receber campos de formulário em vez de JSON, você pode usar `Form`.
+
+!!! info "Informação"
+    Para usar formulários, primeiro instale <a href="https://andrew-d.github.io/python-multipart/" class="external-link" target="_blank">`python-multipart`</a>.
+
+    E.g. `pip install python-multipart`.
```

Comment:
I may be wrong, but Isn't *Ex.* the Portuguese version of *E.g.* ?

Reply:
Done.